packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -0,0 +1,831 @@
+            expect(pageObject.getEditorLeafContents()).toEqual([
+                'user:1',
+                'test:2'
+            ]);
+            await appendUserMentionConfiguration(
+                element,
+                ['user:1'],
+                ['username1']
+            );
+            await appendTestMentionConfiguration(
+                element,
+                ['test:2'],
+                ['testname2']
+            );
+
+            expect(pageObject.getMarkdownRenderedTagNames()).toEqual([
+                'P',
+                `${richTextMentionUsersViewTag}`.toUpperCase(),
+                `${richTextMentionUsersViewTag}`.toUpperCase(),
+                'BR'
+            ]);
+            expect(
+                pageObject.getEditorMentionViewAttributeValues('mention-label')
+            ).toEqual(['username1', 'testname2']);
+            const renderedUserMention = element.lastElementChild as RichTextMentionTest;
+            element.removeChild(renderedUserMention);
+            await waitForUpdatesAsync();
+            expect(pageObject.getMarkdownRenderedTagNames()).toEqual([
+                'P',
+                `${richTextMentionUsersViewTag}`.toUpperCase(),
+                'A'
+            ]);
+            expect(
+                pageObject.getEditorMentionViewAttributeValues('mention-label')
+            ).toEqual(['username1']);
+        });
+
+        it('Should render respective user ID for mention node when removing mapping element from the configuration element', async () => {
+            element.setMarkdown('<user:1> <test:2>');
+
+            expect(pageObject.getEditorTagNames()).toEqual(['P', 'A', 'A']);
+            expect(pageObject.getEditorLeafContents()).toEqual([
+                'user:1',
+                'test:2'
+            ]);
+            await appendUserMentionConfiguration(
+                element,
+                ['user:1'],
+                ['username1']
+            );
+            await appendTestMentionConfiguration(
+                element,
+                ['test:2'],
+                ['testname2']
+            );
+
+            expect(pageObject.getMarkdownRenderedTagNames()).toEqual([
+                'P',
+                `${richTextMentionUsersViewTag}`.toUpperCase(),
+                `${richTextMentionUsersViewTag}`.toUpperCase(),
+                'BR'
+            ]);
+            expect(
+                pageObject.getEditorMentionViewAttributeValues('mention-label')
+            ).toEqual(['username1', 'testname2']);
+            const renderedUserMention = element.lastElementChild as RichTextMentionTest;
+            const renderedMappingUser = renderedUserMention.firstElementChild as MappingUser;
+            renderedUserMention.removeChild(renderedMappingUser);
+            await waitForUpdatesAsync();
+            expect(pageObject.getMarkdownRenderedTagNames()).toEqual([
+                'P',
+                `${richTextMentionUsersViewTag}`.toUpperCase(),
```

Comment:
I'm not sure why the template insertion is needed, it could just be `richTextMentionUsersViewTag.toUpperCase()`

But even then can you just make a constant at the top of this file so this can be done one, like:
```
const RICH_TEXT_MENTION_USERS_VIEW_TAG = richTextMentionUsersViewTag.toUpperCase()
```

Reply:
Updated